docs/usage/tutorials.md

```diff
@@ -0,0 +1,589 @@
+
+You will need internet access and at least 1.5 GB of hardrive space.
+
+#### Software
+
+The tutorial assumes you are on a Unix based operating system, and have already installed Nextflow as well a software environment system such as [Conda](https://docs.conda.io/en/latest/miniconda.html), [Docker](https://www.docker.com/), or [Singularity/Apptainer](https://apptainer.org/).
+The tutorial will use Docker, however you can simply replace references to `docker` with `conda`, `singularity`, or `apptainer` accordingly.
+
+#### Data
+
+First we will make a directory to run the whole tutorial in.
+
+```bash
+mkdir taxprofiler-tutorial
+cd taxprofiler-tutorial/
+```
+
+We will use very small short-read (pre-subset) metagenomes used for testing.
+nf-core/taxprofiler accepts FASTQ or FASTA files as input formats, however we will use FASTQ here as the more common format in taxonomic classification.
+You can download these metagenomes with the following command.
+
+```bash
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474932_ERR5766176_1.fastq.gz
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474932_ERR5766176_2.fastq.gz
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474932_ERR5766176_B_1.fastq.gz
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/fastq/ERX5474932_ERR5766176_B_2.fastq.gz
+```
+
+In this tutorial we will demonstrate running with three different profilers, and in one of those cases, running the same database twice but with different parameters.
+The database consists of two genomes of species known to be present in the metagenomes.
+You can download the databases for Kraken2, Centrifuge, and Kaiju with the following commands.
+
+```bash
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/database/kraken2/testdb-kraken2.tar.gz
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/database/centrifuge/test-db-centrifuge.tar.gz
+curl -O https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/data/database/kaiju/kaiju.tar.gz
+```
+
+To demonstrate that nf-core/taxprofiler can also accept databases as uncompressed folders, we can extract one of them.
+
+```bash
+tar -xzf kaiju.tar.gz
+```
+
+:::note
+You have provide these databases pre-built to the pipeline, nf-core/taxprofiler neither comes with default databases not can generate databases for you.
+For guidance on how to build databases, see the [Retrieving databases or building custom databases](#retrieving-databases-or-building-custom-databases) tutorial.
+:::
+
+Finally, an important step of any metagenomic classification is to remove contamination.
+Contamination can come from many places, typically from the the host of a host-associated sample, however this can also come from laboratory processing samples.
```

Comment:
```suggestion
Contamination can come from many places, typically from the host of a host-associated sample, however this can also come from laboratory processing samples.
```